src/main/groovy/edu/oregonstate/mist/catalogapitest/core/Course.groovy

```diff
@@ -0,0 +1,99 @@
+package edu.oregonstate.mist.catalogapitest.core
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+import javax.xml.bind.annotation.XmlRootElement
+
+@XmlRootElement
+class Course {
+
+    private Integer cid
+    private Integer crn
+    private String courseName
+    private String instructor
+    private String day
+    private String time
+    private String location
```

Comment:
in groovy, getters and setters are created implicitly for properties without access modifiers (public, private, protected).

you should be able to remove `private` from the properties above and then delete the corresponding getters and setters below.

for comparison: https://github.com/osu-mist/dropwizardtest/blob/master/src/main/groovy/edu/oregonstate/mist/dropwizardtest/core/Job.groovy